test/jdk/java/nio/file/attribute/UserDefinedFileAttributeView/Basic.java

```diff
@@ -251,16 +251,31 @@ public void run() throws IOException {
             // Make sure the directory exists.
             Files.createDirectory(longPath);
 
+            String[] elts = longPath.toString().split(":");
+            //String uncName = "\\\\localhost\\" + elts[0] + "$" + elts[1];
+            String host = java.net.InetAddress.getLocalHost().getHostName();
+            String uncName = "\\\\" + host + "\\" + elts[0] + "$" + elts[1];
+            Path uncPath = Path.of(uncName);
```

Comment:
We might have to skip testing UNCs because it will be hard-to-impossible to be reliable in all environments.